js/qz-tray.js

```diff
@@ -2595,11 +2602,15 @@ var qz = (function() {
              *
              * @param {Function|AsyncFunction|Promise<string>} promiseHandler Either a function that will be used as a promise resolver (of format <code>Function({function} resolve, {function}reject)</code>),
              *     an async function, or a promise. Any of which should return the public certificate via their respective <code>resolve</code> call.
-             *
+             * @param {Object} [options] Configuration options for the certificate resolver
+             *  @param {boolean} [options.rejectOnFailure=[false]] Overrides default behavior to call resolve with a blank certificate on failure.
              * @memberof qz.security
              */
-            setCertificatePromise: function(promiseHandler) {
+            setCertificatePromise: function(promiseHandler, options) {
                 _qz.security.certHandler = promiseHandler;
+                if(options && options.rejectOnFailure) {
+                    _qz.security.rejectOnCertFailure = options.rejectOnFailure;
+                }
```

Comment:
```suggestion
                _qz.security.rejectOnCertFailure = options && options.rejectOnFailure;
```

@akberenz should we switch this so that it resets every call?  I was thinking and it may be unobvious if this setting is sticky and remains its last value when it's omitted on subsequent calls.

Reply:
Yeah, I think that change would be for the best from a usability view.